src/include/ipc/dai-intel.h

```diff
@@ -17,6 +17,7 @@
 #define __IPC_DAI_INTEL_H__
 
 #include <ipc/header.h>
```

Comment:
@slawblauciak please add in the commit message more information about multi dai. 

You have added some information in the PR description:

"MultiDAI is a new logical DAI type responsible for allocation of multiple physical child DAI and transparently managing them, routing the audio data."

This must be present in the commit message because PR description will most likely be buried in the dark corners of github.

Reply:
@slawblauciak also use a verb in the commit message. Like: "Introduce multi DAI abi".